### PR TITLE
Refactor: adapt greenbone-nvt-sync to new feed structure

### PR DESCRIPTION
**Pull request by nichtsfrei**
Adapt greenbone-nvt-sync to new feed structure

SC-556
SC-567

### CODE REVIEW

File: tools/greenbone-nvt-sync.in [hunk 1]
```diff
@@ -522,6 +544,7 @@ do_help () {
   echo " --help         display this help"
   echo " --identify     display information"
   echo " --nvtdir dir   set dir as NVT directory"
+  echo " --notusdir dir   set dir as NOTUS directory"
```
Comment (cfi-gb) [inline tools/greenbone-nvt-sync.in:547]:
```suggestion
  echo " --notusdir dir set dir as NOTUS directory"
```
To have the text aligned with the other entries below and above.

**Review by cfi-gb — CHANGES_REQUESTED** [review]

File: tools/greenbone-nvt-sync.in (hunk 1 above)
Comment (nichtsfrei) [reply tools/greenbone-nvt-sync.in]: Thanks, adapted.

File: tools/greenbone-nvt-sync.in [hunk 2]
```diff
@@ -106,6 +117,8 @@ then
   NVT_DIR="@OPENVAS_NVT_DIR@"
 fi
 
+[ -z "$NOTUS_DIR" ] && NOTUS_DIR="/var/lib/notus"
```
[the remark this answers is not in the document]
Comment (nichtsfrei) [reply tools/greenbone-nvt-sync.in]: Changed it to use `@LOCALSTATEDIR@` to have the same effect without having to introduce a new cmake variable.

[the remark this answers is not in the document]
Comment (cfi-gb) [reply tools/greenbone-nvt-sync.in]: Ah, this indeed makes sense. Feel free to mark this conversation as solved :+1:

**Review by ArnoStiefvater — COMMENTED** [review]
When calling `greenbone-nvt-sync --verbose --nvtdir /some/empty/folder --notusdir /some/empty/folder2` it does not use the folders but the defaults. In neither of those folders are the expected files.

**Review by ArnoStiefvater — APPROVED** [review]

**Pull request merged**